src/psyclone/psyir/transformations/acc_update_trans.py

```diff
@@ -227,20 +223,15 @@ def _add_update_directives(self, node_list):
                     inputs.add(Signature(symbol_name))
                     outputs.add(Signature(symbol_name))
 
-        # Copy any data that is read by this region to the host (resp. device)
-        # if it is on the device (resp. host).
+        # Copy any data that is accessed by this host region to the host
+        # (resp. device) if it is on the device (resp. host).
+        global IN, OUT
```

Comment:
Could you rename these - perhaps `HOST_INPUT` and `HOST_OUTPUT`?

Reply:
Failing that, please extend the comment along the lines of what you've put in L343-344.

Reply:
Done.